notebooks/Training_a_LoRA_With_Instruct_Lab.ipynb

```diff
@@ -79,7 +79,7 @@
       "outputs": [],
       "source": [
         "# installing dependencies\n",
-        "!pip install -q -U transformers accelerate peft datasets bitsandbytes trl"
+        "!pip install -q -U transformers accelerate peft datasets bitsandbytes trl pyarrow==14.0.1 requests==2.31.0 torch==2.3.0"
```

Comment:
Can you elaborate on the issue the dep pin is fixing? Thanks!

Reply:
because I was trying to avoid this:
![image](https://github.com/instructlab/instructlab/assets/3435696/9e2bbc0b-0590-4e6c-b557-b490e58aae5a)
which is the default landing to the notebook.